app/Services/GitHubClientGapTracker.php

```
@@ -210,10 +211,11 @@ private function testCheckStatusCapabilities(string $owner, string $repo, int $p
 
             // Test status checks endpoint
             try {
-                if (method_exists(Github::class, 'commits')) {
+                // Check if commit resource exists and has status method
+                if (method_exists(Github::class, 'commits') && method_exists(Github::commits(), 'status')) {
                     $statuses = Github::commits()->status($owner, $repo, $headSha);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$statuses'. (Unused Code Rules)

(UnusedLocalVariable)